frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -286,6 +333,29 @@ pub mod pallet {
 			T::PalletId::get().into_account()
 		}
 
+		#[transactional]
+		fn set_access(account_id: T::AccountId, access: AccessRights) -> DispatchResult {
```

Comment:
Finally, we don't need to take the argument (`account_id`) by value (see above comments).

Reply:
Resolved!